provider/common/helpers.go

```diff
@@ -151,3 +151,12 @@ func HandleErr(res *ocmerrors.Error, err error) error {
 	errType := weberr.ErrorType(res.Status())
 	return errType.Set(errors.Errorf("%s", msg))
 }
+
+// StringValue returns the value of the string pointer passed in or
+// "" if the pointer is nil.
```

Comment:
If I'm not wrong - that function is not needed anymore...

Reply:
You right, removed it